felix/bpf-gpl/conntrack_types.h

```diff
@@ -67,28 +68,30 @@ struct calico_ct_leg {
 #define CT_INVALID_IFINDEX	0
 struct calico_ct_value {
 	__u64 rst_seen;
-	__u64 last_seen; // 8
-	__u8 type;		 // 16
+	__u64 last_seen;	// 8
+	__u8 type;		// 16
 	__u8 flags;
 
+	__u8 flags3;
+	__u8 flags4;
```

Comment:
perhaps `__u16 flags34` ? It is 16bit aligned. But it is a nit.

Reply:
In this case, it was the opposite... comments were aligned with spaces. I converted to tabs - it looks aligned with tabsize == 8spaces, but Github seems to visualise tabs with 4 spaces :thinking: Can return it to how it was if you like.